peachjam/templates/peachjam/_document_content_sidebar.html

```diff
@@ -3,6 +3,9 @@
 contents of #navigation-content will be placed in [data-offcanvas-body] for tablet/mobile screensize
 and #navigation-column for desktop screensize.
 -->
+<div class="d-none d-md-block">
+  <a href="#document-content">{% trans 'Skip table of contents' %}</a>
```

Comment:
I wonder if this should be what we're skipping to, rather than what is being skipped? Or maybe both: "Skip table of contents and go to document content." ?

Reply:
i was having the same debate as to the best wording to use. Lets use the same as the link above, `Skip to document content`